crates/ruff_workspace/src/configuration.rs

```diff
@@ -714,7 +714,9 @@ impl LintConfiguration {
             .collect();
 
         // The fixable set keeps track of which rules are fixable.
-        let mut fixable_set: RuleSet = RuleSelector::All.rules(&preview).collect();
+        let mut fixable_set: RuleSet = RuleSelector::All
+            .rules(&preview.without_require_explicit())
```

Comment:
There is additional logic downstream for `fixable` and `unfixable` that may require a similar change. I need to create some example test cases.

Reply:
Can we use `RuleSelector::All.all_rules()` instead here?

Reply:
That's a good idea.

Should we require explicit codes for fixable and unfixable for preview rules though? Or should the setting only affect select?